got3/manager/TweetManager.py

```diff
@@ -118,9 +118,9 @@ def getJsonReponse(tweetCriteria, refreshCursor, cookieJar, proxy):
 		]
 
 		if proxy:
-			opener = urllib2.build_opener(urllib2.ProxyHandler({'http': proxy, 'https': proxy}), urllib2.HTTPCookieProcessor(cookieJar))
+			opener = urllib.request.build_opener(urllib.ProxyHandler({'http': proxy, 'https': proxy}), urllib.request.HTTPCookieProcessor(cookieJar))

```

Comment:
Hi, could you verify that your function can resolve all references? Reason, why I ask is, that ProxyHandler is actually a class in urllib.request.ProxyHandler, but you are writing urllib.ProxyHandler